src/app/story/directives/basic.component.ts

```diff
@@ -108,8 +112,93 @@ export class BasicComponent implements OnDestroy, DoCheck {
   ) {
     this.tabSub = tab.model.subscribe((s: StoryModel) => {
       this.story = s;
+      this.series = new SeriesModel(null, s.parent);
       this.loadVersionTemplates();
+
+      concat(
+        this.loadDistribution(),
+        this.loadPodcast(),
+        this.loadUnpublished()
+      ).subscribe({complete: () => {
+          if (this.story.isNew && !this.podcast.complete && this.missingDrafts()) {
+            this.alertForDrafts();
+          }
+        }});
+    });
+  }
+
+  missingDrafts() {
+    return (this.hasDrafts !== null && !this.hasDrafts);
+  }
+
+  alertForDrafts() {
+    const msg = `The more drafts you add, the better we can support your podcast.
+     Please add
+     <a href="/series/${this.series.id}/plan">
+       all of your known upcoming episodes
+     </a>
+     to the Production Calendar.`;
+
+    this.modal.alert('Missing Episode Drafts', msg,  () => {  });
+  }
+
+  loadDistribution() {
+    let obs = this.series.loadRelated('distributions');
+
+    obs.subscribe(() => {
+      this.distribution = this.series.distributions.find((d) => d.kind === 'podcast');
+    });
+
+    return obs;
+  }
+
+  loadPodcast() {
+    if (!this.distribution) {
+      this.podcast = null;
+      return of({});
+    }
+
+    let obs = this.distribution.loadRelated('podcast');
+    obs.subscribe(() => {
+      this.podcast = this.distribution.podcast;
     });
+
+    return obs;
+  }
+
+  loadUnpublished() {
+    this.hasDrafts = null;
+
+    const halParams = {
+      page: 1,
+      per: 1,
+      filters: `after=${new Date().toISOString()}`,
+      zoom: false
+    };
+
+    let obs = this.series.doc.followItems('prx:stories', halParams);
+
+    obs.subscribe((stories: HalDoc[]) => {
+      const story = stories[0] ? new StoryModel(this.story.parent, stories[0]) : null;
```

Comment:
One quirk of how HalDoc works ...

If you new a Model, it's by default going to [loadRelated](https://github.com/PRX/publish.prx.org/blob/master/src/app/shared/model/story.model.ts#L145).  Which for the StoryModel fires of 3 additional XHR requests (since you zoom=falsed, those resources weren't embedded).

But since the below check on line 184 is now redundant with the `after=` filter ... you could probably just forgo the StoryModel entirely here.